src/main.js

```diff
@@ -119,6 +119,7 @@ class MiniMediaPlayer extends LitElement {
       source: 'default',
       sound_mode: 'default',
       toggle_power: true,
+      volume_step: 10,
```

Comment:
I would prefer if the option wasn't set by default, so either `0` or `null` here.

Reply:
https://github.com/kalkih/mini-media-player/pull/308/commits/dc6c5f3f1f0be91a4109a88c81ac6a37abd9c9d5